src/coreclr/vm/genanalysis.cpp

```diff
@@ -100,6 +100,7 @@ bool gcGenAnalysisDump = false;
         EP_SESSION_TYPE_FILE,
         EP_SERIALIZATION_FORMAT_NETTRACE_V4,
         false,
+        true,
```

Comment:
Instead of passing true for a lot of calls, would it make sense to remove the parameter and have true as default until there is a need to extend the API?

The same goes to the change to ep_enable, eventpipe_enable_2, it might be better to add ep_enable_3 that have the new stack parameter and call that version in the case where it can be anything different than true, that would dramatically reduce the changes needed in this PR.

Reply:
Perhaps we should replace this large parameter list with a struct?
```
EventPipeAdapter::Enable(ep_session_options* pOptions)
```

It would touch a bunch of code once to do the refactor, but future modifications to pass state through or adjust defaults should touch fewer places.

Reply:
Jepp, that is always an option, if we would like to keep the changes lower then we could have a ep_enable_3 that takes a versioned struct and only align calls that would need to pass the additional parameters. It will bubble up to some degree, but only in places that would need set that parameter (that currently don't hardcode true as done in current PR) and we could do iterative refactoring instead of a complete overhaul in this PR.